src/Service/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -124,18 +124,8 @@ public abstract class BaseSqlQueryStructure : BaseQueryStructure
         /// </summary>
         public Type GetColumnSystemType(string columnName)
         {
-            if (GetUnderlyingSourceDefinition().Columns.TryGetValue(columnName, out ColumnDefinition? column))
```

Comment:
Does the validation that happens for REST happen in Production mode too?

Looking at this function standalone, it seems safe practice to check for the return type of `TryGetValue`. What if someone wants to call this function outside of the current usecase since its `public` - how does it mandate the columnName is guaranteed to exist in the source definition?

Reply:
I agree. Retaining the checks